tests/suites/test_suite_pkwrite.function

```diff
@@ -72,3 +72,32 @@ exit:
     mbedtls_pk_free( &key );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PEM_WRITE_C */
+void pkcs8_write_key_check( char *key_file )
+{
+    mbedtls_pk_context key;
+    unsigned char buf[5000];
+    unsigned char check_buf[5000];
+    int ret;
+    FILE *f;
+    size_t ilen;
+
+    memset( buf, 0, sizeof( buf ) );
+    memset( check_buf, 0, sizeof( check_buf ) );
+
+    mbedtls_pk_init( &key );
+    TEST_ASSERT( mbedtls_pk_parse_keyfile( &key, key_file, NULL ) == 0 );
```

Comment:
I would try adding tests without using file system.

Reply:
Well, I effectively used the existing tests as a template for the new one, so we are using this practice even without this PR. I suppose what I could do is hard code the PEM key into the .data file. Is that better?

Reply:
I wouldn't say this test is not needed, I would say we are missing tests though